frappe/public/js/frappe-web.bundle.js

```diff
@@ -24,3 +24,4 @@ import "./bootstrap-4-web.bundle";
 
 import "../../website/js/website.js";
 import "./frappe/socketio_client.js";
+import "./frappe/form/controls/control.js";
```

Comment:
This is QUITE huge (~1.8MB). Why was this required on website? 


@pateljannat / @surajshetty3416